django_app/redbox_app/redbox_core/views.py

```diff
@@ -155,21 +155,18 @@ def remove_doc_view(request, doc_id: str):
 
 
 @login_required
-def sessions_view(request, session_id: str = ""):
-    USE_STREAMING = False
-    STREAMING_ENDPOINT = "ws://localhost:8888"
-
+def sessions_view(request: HttpRequest, session_id: str = ""):
     chat_history = ChatHistory.objects.all().filter(users=request.user)
 
     messages = []
     if session_id:
         messages = ChatMessage.objects.filter(chat_history__id=session_id)
-
+    endpoint = URL.build(scheme="ws", host=request.get_host(), path=settings.STREAMING_ENDPOINT)
```

Comment:
Should the endpoint change depending on whether settings.USE_STREAMING is true or not?

Reply:
If streaming is False, we don't use the endpoint at all.

Reply:
Maybe it's worth a comment explaining that?